bedrock/mozorg/redirects.py

```diff
@@ -271,14 +258,6 @@ def decider(request, **kwargs):
     # bug 957664
     redirect(r'^press/awards(?:/|\.html)?$', 'https://blog.mozilla.org/press/awards/'),
 
-    # bug 987059, 1050149, 1072170, 1208358
-    redirect(r'^ja/about/?$', 'https://www.mozilla.jp/about/mozilla/'),
-    redirect(r'^ja/about/japan/?$', 'https://www.mozilla.jp/about/japan/'),
```

Comment:
Are we ok that `/ja/about/japan/` will now 404? I'm not entirely clear from [bug 1208358](https://bugzilla.mozilla.org/show_bug.cgi?id=1208358) on the history for why this redirect existed.